Updates for fo76 and adding bsa loading for fo76

eckserah opened the pull request.

Commit:
Some fixes for FO76

Fix some spacing issues with the archtype checking.
Add Dialogue Subtype Enum for DIAL/SNAM.
Fix Subtype Names.
Fix DIAL Categories
Fix ETLG location in ENTM
FNAM, GNAM, Possibly ENAM in IMGS are all the same with slight changes.

Commit:
Merge branch 'dev-4.1.4' of https://github.com/TES5Edit/TES5Edit into dev-4.1.4

Commit:
Add fallout76 to the BSA loading

Infernio commented on wbDefinitionsFO76.pas:
```diff
@@ -12672,8 +12800,30 @@ procedure DefineFO76g;
+      wbFloat('Middle Gray'),
+      wbFloat('Unknown')
+    ], cpNormal),
     wbArray(FNAM, 'Unknown', wbFloat('Unknown')),
```
Are there really two FNAMs right after each other? Or did you just forget to delete the old definition?

eckserah replied:
Nope I forgot to remove it. Fixing thank you

eckserah replied:
fixed in the commit below

Commit:
Additions and changes for FO76 version 1.2.6.4

add AMMR, AMRS, ATTR, ESCS, PCDP, PERM, PERT, QARF, QRLI, QRLR, QRRI, RCND, XACR, XPSL, XRTM, XSED.
Define deciders for telekinesis and grow flora.
Fix extra IMGS\FNAM and define gnam and enam
Update a bunch of the Extra data records.
Some naming clarifications.

Commit:
Fix EXPL/DATA for both FO76 and FO4

Commit:
Add size decider functions and fix up some FO76 Data structures.
